src/components/ABVCalculator.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <form id="abvCalc" class="max-w-48 flex flex-col">
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit ABV in %
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="spiritABV"
+        id="ABV"
+        placeholder="37.5"
+      />
+    </label>
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit quantity in ml
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="spiritQuantity"
+        id="alcoholQuantity"
+        placeholder="50"
+      />
+    </label>
+    <label class="justify-center w-full flex flex-col my-4"
+      >Mixer quantity in ml
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="mixerQuantity"
+        id="mixerQuantity"
+        placeholder="330"
+      />
+    </label>
+  </form>
+
+  <div class="results">
+    <p>Your drink is <span :class="computedABVColourClasses" class="font-bold">{{ `${calculateTotalDrinkABV}% ABV` }}</span>
+    </p>
+    <p>
+      There is <span class="font-bold" :class="computeTotalAlcoholColourClasses">{{ calculateTotalAlcoholQuantity }}ml</span> of alcohol in your drink
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      spiritABV: 37.5,
+      spiritQuantity: 50,
+      mixerQuantity: 330,
+    };
+  },
+  computed: {
+    abvToDecimal() {
+      return this.spiritABV / 100;
+    },
+    calculateTotalAlcoholQuantity() {
+      return this.spiritQuantity * this.abvToDecimal;
+    },
+    totalDrinkQuantity() {
+      return this.mixerQuantity + this.spiritQuantity;
+    },
+    calculateTotalDrinkABV() {
+      if (this.totalDrinkQuantity == 0 ){return 0}
+      if (this.totalDrinkQuantity != 0 ){return ((this.calculateTotalAlcoholQuantity / this.totalDrinkQuantity)*100).toFixed(2);
+    }},
+    computedABVColourClasses(){
+    
+      if (this.calculateTotalDrinkABV >=2.8 && this.calculateTotalDrinkABV < 6){
+        return "text-green-600";
+      }
+
+      if (this.calculateTotalDrinkABV >= 6 && this.calculateTotalDrinkABV < 10){
+        return "text-amber-400";
+      }
+       if (this.calculateTotalDrinkABV >= 10){
+        return "text-red-600";
+      }
+      
+        return "text-green-400";

```

Comment:
This can never be reached

Reply:
Oh yeah this is true because less than 2.8 is also less than 10 and returns the amber styles